catroid/src/main/java/org/catrobat/catroid/ProjectManager.java

```diff
@@ -243,6 +243,14 @@ public void loadProject(String projectName, Context context) throws LoadingProje
 				project.setCatrobatLanguageVersion(0.992f);
 			}
 			if (project.getCatrobatLanguageVersion() == 0.992f) {
+				project.updateCollisionFormulasToNewVersion();
+				project.setCatrobatLanguageVersion(0.993f);
+				asynchronousTask = false;
+				saveProject(context);
+				loadProject(projectName, context);
```

Comment:
As this is bad for the performance (especially for big programs): Do you really need to save and load the program again?